src/styles/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  top: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: $main-font;
+}
+
+.nav {
+  flex-direction: row;
+  margin-right: 50px;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    padding: 0 40px;
+    margin: 16px -50px;
+  }
+
+  &__item {
+    display: inline-block;
+    list-style-type: none;
+    padding: 3px 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $link-color;
+    font-family: $main-font;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+    padding: 23px 0;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order